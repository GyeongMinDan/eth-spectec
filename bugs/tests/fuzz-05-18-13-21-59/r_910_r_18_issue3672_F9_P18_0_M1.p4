control c()() {
     action a(in bit<1> v1, in list<_> v2) {
    }
     apply {
 }
   }
