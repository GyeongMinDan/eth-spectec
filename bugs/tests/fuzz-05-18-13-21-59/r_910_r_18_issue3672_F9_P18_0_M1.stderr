likely_hits/r_910_r_18_issue3672_F9_P18_0_M1.p4(1): [--Wwarn=unused] warning: Control c is not used; removing
control c()() {
        ^
[--Wwarn=missing] warning: Program does not contain a `main' module
